jsx/Form.js

```diff
@@ -780,7 +780,7 @@ class TextboxElement extends Component {
   }
 
   handleChange(e) {
-    this.props.onUserInput(this.props.name, e.target.value);
+    this.props.onUserInput(this.props.name, e.target.value, this.props.id, 'textbox');
```

Comment:
These two changes in `Form.js` are what allow the `onElementUpdate` function within the `FilterForm` component to accept `id` and `type`. There is likely a way of doing this that doesn't necessitate changing Form.js, but I'm thinking that it may be useful to be able to access the `id` and `type` of components `onUserInput` loris-wide.